packages/x-date-pickers/src/models/value.ts

```diff
@@ -0,0 +1,6 @@
+import { PickerValidDate } from './pickers';
+
+export type SimpleValue = PickerValidDate | null;
```

Comment:
SimpleValue sounds somewhat strange. 🤷 
Have you considered `PickerValue`?
```suggestion
export type PickerValue = PickerValidDate | null;
```

Reply:
I would like to rename `DateRange` to have both interfaces being named coherently.
I was going for `SimpleValue` / `RangeValue`, but we could go for `PickerValue` / `PickerRangeValue` if you prefer.

Reply:
My vote goes for the `PickerValue` and `PickerRangeValue`. 👍 
WDYT @arthurbalduini?